hw/ip/adc_ctrl/dv/cov/adc_ctrl_cov.core

```diff
@@ -3,15 +3,16 @@ CAPI=2:
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 name: "lowrisc:dv:adc_ctrl_cov"
-description: "LC_CTRL functional coverage and bind files"
+description: "ADC_CTRL functional coverage and bind files"
 filesets:
   files_rtl:
     depend:
-      - lowrisc:ip:lc_ctrl
+      - lowrisc:ip:adc_ctrl
 
   files_dv:
     depend:
       - lowrisc:dv:dv_utils
+      - lowrisc:ip:adc_ctrl
```

Comment:
looks like this core file get imported twice?

Reply:
Removed the second one.